src/app/ApplicationTemplate.Shared.Views/Framework/Launcher/AppStoreUriProvider.cs

```diff
@@ -0,0 +1,37 @@
+﻿using System;
+using Microsoft.Extensions.Configuration;
+
+namespace ApplicationTemplate;
+
+/// <summary>
+/// Implementation of the <see cref="IAppStoreUriProvider"/>.
+/// </summary>
+public sealed class AppStoreUriProvider : IAppStoreUriProvider
+{
+	private readonly IConfiguration _configuration;
+
+	/// <summary>
+	/// Initializes a new instance of the <see cref="AppStoreUriProvider"/> class.
+	/// </summary>
+	/// <param name="configuration"> The configuration where we get the Uris. </param>
+	public AppStoreUriProvider(IConfiguration configuration)
+	{
+		_configuration = configuration;
+	}
+
+	/// <inheritdoc />
+	public Uri GetAppStoreUri()
+	{
+		var mockOptions = _configuration.GetSection("ApplicationStoreUrls").Get<ApplicationStoreUrisOptions>();
+		Uri uri;
+
+#if ANDROID
```

Comment:
Use `__ANDROID__` and `__IOS__` to make consistent with documentation and the whole application code.